sdcm/nemesis.py

```diff
@@ -746,6 +747,18 @@ def disrupt_validate_hh_short_downtime(self):
         assert self.tester.hints_sending_in_progress() is False, "Hints are sent too slow"
         self.tester.verify_no_drops_and_errors(starting_from=start_time)
 
+    def disrupt_snapshot_operations(self):
+        result = self._run_nodetool('nodetool snapshot', self.target_node)
+        self.log.info(result)
+        snapshot_name = re.findall('(\d+)', result.stdout, re.MULTILINE)[0]
+        result = self._run_nodetool('nodetool listsnapshots', self.target_node)
+        if snapshot_name in result.stdout and not result.stderr:
+            self.log.info('Snapshot %s created' % snapshot_name)
+        else:
+            self.log.info(result)
```

Comment:
if this is the condition when snapshot was not created, I think we need to assert, since it is an issue

Reply:
Adding avocado test.fail() method. It add approriate messages to log